app/code/community/Drip/Connect/Model/Configuration.php

```diff
@@ -66,6 +66,16 @@ public function getStoreId()
         return $this->storeId;
     }
 
+    /**
+     * The website ID attached to the current store view.
+     *
+     * @returns int
```

Comment:
I'm sorry, I messed up with my comment.
```suggestion
     * @return int
```

Reply:
I always mess this up, and then wonder why docblock syntax highlighting isn't working…